mise.toml

```diff
@@ -10,9 +10,9 @@ yq = "4.44.5"
 # Foundry is a special case because it supplies multiple binaries at the same
 # GitHub release, so we need to use the aliasing trick to get mise to not error
 # The git ref here should be on the `stable` branch.
-forge = "v1.0.0"
-cast = "v1.0.0"
-anvil = "v1.0.0"
+forge = "nightly"
+cast = "nightly"
+anvil = "nightly"
```

Comment:
We should pin to a commit hash, e.g. `nightly-fullCommitHashHere`

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/664/commits/c908437b9e807b01b16c9867d5f9b15aa62d615b